examples/controls_example/public/app/react_control_example/react_control_example.tsx

```diff
@@ -139,7 +139,6 @@ export const ReactControlExample = ({
       addNewPanel: () => {
         return Promise.resolve(undefined);
       },
-      lastUsedDataViewId: new BehaviorSubject<string>(WEB_LOGS_DATA_VIEW_ID),
```

Comment:
I wonder if we should add `dataViews` to the parent to invoke this code in the example?